include/router_device_worker.hrl

```diff
@@ -1,3 +1,17 @@
+-define(DOWNLINK_REGIONS, [
+    'US915',
+    'AU915',
+    'EU868',
+    'CN470',
+    'AS923_1',
+    'AS923_2',
+    'AS923_3',
+    'AS923_4'
+]).
+
+-type 'downlink.region'() ::
```

Comment:
```suggestion
-type downlink_region() ::
```
Do you mind if we do this instead? We have an aversion against `'` on atoms

Reply:
Addressed.